begin/cmdline.py

```diff
@@ -58,10 +58,16 @@ def from_name(self, name, default=NODEFAULT, section=None):
         "Get default value from argument name"
         if len(self._parser.sections()) > 0:
             section = self._section if section is None else section
-            try:
-                default = self._parser.get(section, name)
-            except (configparser.NoSectionError, configparser.NoOptionError):
-                pass
+            if isinstance(section, list):
+                sections = section
+            else:
+                sections = []
+                sections.append(section)
+            for section in sections:
```

Comment:
I'm not sure I agree with this behavior. 
We need to find a way to make this more deterministic in my opinion.
WDYT?